cert-manager/src/index.tsx

```diff
@@ -0,0 +1,154 @@
+import { registerRoute, registerSidebarEntry } from '@kinvolk/headlamp-plugin/lib';
+import { CertificateRequestDetail } from './components/certificateRequests/Detail';
+import { CertificateRequestsList } from './components/certificateRequests/List';
+import { CertificateDetail } from './components/certificates/Detail';
+import { CertificatesList } from './components/certificates/List';
+import { ChallengeDetail } from './components/challenges/Detail';
+import { ChallengesList } from './components/challenges/List';
+import { ClusterIssuerDetail } from './components/clusterIssuers/Detail';
+import { ClusterIssuersList } from './components/clusterIssuers/List';
+import { IssuerDetail } from './components/issuers/Detail';
+import { IssuersList } from './components/issuers/List';
+import { OrderDetail } from './components/orders/Detail';
+import { OrdersList } from './components/orders/List';
+
+// Main Cert-manager sidebar entry
+registerSidebarEntry({
+  name: 'Cert-manager',
+  url: '/cert-manager/certificates',
+  icon: 'mdi:certificate',
+  parent: '',
+  label: 'cert-manager',
+});
+
+// 1. certificates.cert-manager.io
+registerSidebarEntry({
```

Comment:
this pattern repeats here, `registerSidebarEntry` + `registerRoute` for Details and List could be put in a function